gdcdictionary/schemas/analyte.yaml

```diff
@@ -81,30 +97,35 @@ properties:
       - W
       - X
       - Y
-  amount:
+  analyte_volume:
     term:
-      $ref: "_terms.yaml#/amount"
+      $ref: "_terms.yaml#/analyte_volume"
     type: number
   concentration:
     term:
       $ref: "_terms.yaml#/concentration"
     type: number
-  a260_a280_ratio:
+  normal_tumor_genotype_snp_match:
     term:
-      $ref: "_terms.yaml#/a260_a280_ratio"
-    type: number
-  well_number:
-    term:
-      $ref: "_terms.yaml#/well_number"
-    type: string
+      $ref: "_terms.yaml#/normal_tumor_genotype_snp_match"
+    enum:
+      - "Yes"
```

Comment:
This is a workaround done because without the quotes the yes and no are interpreted as true/false.